internal/crypto/internal_ca.go

```diff
@@ -0,0 +1,160 @@
+package crypto
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"errors"
+	"math/big"
+	"os"
+	"time"
+
+	oscrypto "github.com/openshift/library-go/pkg/crypto"
+)
+
+type internalCA struct {
+	Config          *TLSCertificateConfig
+	SerialGenerator oscrypto.SerialGenerator
+}
+
+func ensureInternalCA(certFile, keyFile, serialFile, subjectName string, expireDays int) (CABackend, bool, error) {
```

Comment:
can you add some go docs on this specifically what the bool represents

Reply:
History. 
I did not change this - it is inherited from the original CA code and signifies was the CA loaded or generated on the fly.